collectors/ebpf_process.plugin/ebpf_process.c

```diff
@@ -529,26 +529,26 @@ static void move_from_kernel2user_global() {
         }
     }
 
-    aggregated_data[0].call = res[0]; //open
-    aggregated_data[1].call = res[14]; //close
-    aggregated_data[2].call = res[8]; //unlink
-    aggregated_data[3].call = res[5] + res[21]; //read + readv
-    aggregated_data[4].call = res[2] + res[18]; //write + writev
-    aggregated_data[5].call = res[10]; //exit
-    aggregated_data[6].call = res[11]; //release
-    aggregated_data[7].call = res[12]; //fork
-    aggregated_data[8].call = res[16]; //thread
+    aggregated_data[0].call = res[NETDATA_KEY_CALLS_DO_SYS_OPEN]; //open
```

Comment:
I think we can kick out those comments now as we use Defines not magic numbers anymore.

Reply:
I removed the comments, I liked your suggestion. Thank you!